src/components/Book.js

```diff
@@ -0,0 +1,37 @@
+import { CircularProgressbar } from 'react-circular-progressbar';
+import classes from './Books.module.css';
+import 'react-circular-progressbar/dist/styles.css';
+
+const percentage = 66;
+const Book = () => (
+  <>
+    <div className={classes.bookCard}>
+      <div className={classes.bookInfo}>
+        <div>
+          <p className={classes.catagory}>The Hunger Games</p>
+          <h3 className={classes.title}>Suzanne Collins</h3>
+          <p className={classes.author}>Action</p>
```

Comment:
- [ ] Hi, according to the requirements; this component should use props to pass the book details. please use props for the book details instead of hard coding them.